package.xml

```diff
@@ -64,7 +59,6 @@
   <run_depend>tf2_geometry_msgs</run_depend>
   <run_depend>xacro</run_depend>
 
-  <test_depend>moveit_resources_panda_moveit_config</test_depend>
```

Comment:
I thought this was needed for the [tests](https://github.com/ros-planning/moveit2_tutorials/tree/main/doc/tests) tutorial?? We can remove it if that's not the case.

Reply:
I thought it would be superseded by the `run_depend`